### ISSUE TITLE
Error when using 8MHz internal Oscillator on m88

### ISSUE THREAD

**Issue by pcfreak1201** [nerdralph/ArduinoShrink#2]
In the description, you write

> ArduinoShrink supports only ATmega8 and ATmegax8 MCUs. Accurate timing for millis() and delay() requires a freqquency that is a power of two such as 1Mz, 8Mhz, or 16Mhz. 

But using 8MHz internal Oscillator produces an error because of
```
#if F_CPU != 16000000
#error Only 16Mhz F_CPU supported
#endif
```

What is right?

**Comment by nerdralph** [comment]
I forgot to remove that from an earlier version.  It's fixed now.

**Comment by nerdralph** [comment]
fixed with commit 583dba5b15949265eaab81012176097e9b9f8e31
